portfolio/src/main/webapp/about.js

```diff
@@ -0,0 +1,93 @@
+let answer = 0;
+
+function loadInTruth() {
+
+    let firstTruth = Math.floor(Math.random() * truths.length);
+    let secondTruth = Math.floor(Math.random() * truths.length);
```

Comment:
These are just actually indices right? Suggest naming them something a little more accurate. (maybe throw "index" in there)

Reply:
Renamed truth and lie varaibles to make more clear